src/NHibernate.Test/NHSpecificTest/NH3374/FixtureByCode.cs

```diff
@@ -15,9 +13,11 @@ protected override HbmMapping GetMappings()
 			mapper.Class<Document>(rc =>
 			{
 				rc.Id(x => x.Id, idMapper => idMapper.Generator(Generators.Identity));
-				rc.ManyToOne(x => x.Blob, m =>
+				rc.ManyToOne(x => x.Blob,
+					m =>
 					{
 						m.Cascade(Mapping.ByCode.Cascade.All);
+						m.Column("`Blob`");
```

Comment:
I think the correct <tm> way to do this is to mark "blob" as [a keyword for Firebird](http://www.firebirdsql.org/refdocs/langrefupd25-reskeywords-full-keywords.html) dialect.